node/src/chain_spec.rs

```diff
@@ -32,6 +34,9 @@ pub type ChainSpec = GenericChainSpec<GenesisConfig>;
 const FFNET_CHAIN_SPEC: &[u8] = include_bytes!("./chain_spec/ffnet.json");
 const LATEST_RUNTIME_WASM: &[u8] = include_bytes!("../../runtime/latest.wasm");
 
+/// Chain for local development with an isolated node.
+///
+/// If `runtime` is given it is used as the genesis runtime. Uses dummy PoW that does not mine.
```

Comment:
```suggestion
/// If `runtime` is given, it is used as the genesis runtime. Uses dummy PoW that does not mine.
```

One question here. If we are operating on a single node and it doesn't mine, wouldn't that mean that txs are never really included in a block?

Reply:
I’ve used the wrong word “mine” here. I’ve updated the comment to clarify it.